actions-server/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": false
+}
```

Comment:
@namoscato not sure if we want to move this to top level directory, or that each js app should have it's own..

Reply:
I'd be in favor of bumping it to the root which would also simplify IDE configuration. Then, I think we would just have to [explicitly reference](https://prettier.io/docs/en/cli.html#--find-config-path-and---config) the configuration file in yarn scripts.

It looks like we'd want to add `prettier` as a dev dependency in here as well.